examples/basecoin/app/app.go

```diff
@@ -75,6 +72,9 @@ func NewBasecoinApp(logger log.Logger, db dbm.DB) *BasecoinApp {
 		AddRoute("ibc", ibc.NewHandler(ibcMapper, coinKeeper)).
 		AddRoute("simplestake", simplestake.NewHandler(stakeKeeper))
 
+	// Define the feeHandler.
+	app.feeHandler = func(ctx sdk.Context, tx sdk.Tx, fees sdk.Coins) {}
```

Comment:
needs implementation or a new Issue, same with democoin.

Reply:
Yes, will be implemented by the `stake` module, I guess this PR can just wait for that, no reason to merge beforehand

Reply:
That's what this does at the moment (the fees are burned and the total supply is decreased).

Reply:
Did you want the handler to do something else (log the burn somehow)?

Reply:
nah I think that's fine - will review one last time and then merge soon unless something comes up